### DISCUSSION TITLE
Many files in PR makes Files tab unusable

### DISCUSSION THREAD

**Discussion by osdiab** [community/community#5670] [Projects and Issues]
Yarn is a popular tool in the JS world, and they released Zero-Installs, a paradigm where you check in dependencies into your repo. When you do so you can end up with diffs including large numbers of ZIP files.

You can filter the files in a PR by file extension already, but  when you do those files still get rendered wherever they are - this causes a couple issues

- if there are a lot of files, you still have to scroll a ton to look at code
- it can also cause serious performance issues where my browser locks up due to the large number of files
- the files from `yarn` are placed in a directory called `.yarn`, which due to the default sorting of files causes your code to generally come after all the meaningless files

This makes it excruciating to review PRs with these files in them - sometimes I can't even get to the code to review.

I would suggest an option to make hiding file extensions, not just collapse the files, but actually remove them from the files list.

**Comment by lfolco** [comment]
I keep seeing this issue reported over and over, yet nothing is ever done about it. Any updates?
